packages/neuvector/package.yaml

```diff
@@ -1,5 +1,5 @@
-url: https://neuvector.github.io/neuvector-helm/core-2.2.0-b1.tgz
-version: 100.0.0
+url: https://neuvector.github.io/neuvector-helm/core-2.2.0-b2.tgz
+version: 100.0.1
```

Comment:
No need to bump this, if `-bX` is a pre-release version, only the pre-release is bumped. the 100.x.x version is bumped for each Rancher chart's release (100.0.1 would be the version to release in 2.6.6).